nototools/generate_website_2_data.py

```diff
@@ -59,6 +58,13 @@
 APACHE_LICENSE_LOC = path.join(FONTS_DIR, 'LICENSE')
```

Comment:
Akk.  Yeah, this predates the switch to use OFL.

Currently, all the fonts use the OFL license, but if any did use Apache they'd get the wrong one, so I still need to fix this.  I'll use the one at the top of noto-emoji as you suggest.

noto-emoji is weird, the code is Apache but the fonts are OFL.  We have a different license in the fonts subdirectory than at the top level...